README.md

```diff
@@ -280,11 +280,11 @@ Under the `Infrastructure` code folder, create an Assembly Definition also named
 
 ![Infrastructure Assembly Definition](https://deimors.github.io/UnityDependencyInjection/Images/Infrastructure%20Assembly%20Definition.png)
 
-This assembly definition causes two errors occur for the `SceneInstaller` class because the newly defined assembly does not reference the Zenject assembly. The Zenject plugin folder also contains an assembly definition, named `zenject`, and so a reference can be added from the `Infrastructure` project to the `zenject` project in order to handle these errors.
+This assembly definition causes two errors to occur for the `SceneInstaller` class because the newly defined assembly does not reference the Zenject assembly. The Zenject plugin folder also contains an assembly definition, named `zenject`, and in the `Infrastructure` project add an Assembly Definition Reference to the `zenject` project in order to handle these errors. In the current version of Unity, you must select a different file in the Project tree for the Assebmly Definition changes to be applied. If you are prompted to apply the "Unapplied import settings", hit the Apply button.
```

Comment:
Actually, if you scroll down on the Assembly Definition inspector panel, at the bottom there is an "Apply" button which will apply your changes. The steps you mention is the "in case you forgot to hit Apply" way, which is viable, but probably better to just point out the normal way.

Also, just a small typo in the change, s/Assebmly/Assembly/

Reply:
Sweet, I never even noticed that small `Apply` button. Content updated 👍

Reply:
```suggestion
This assembly definition causes two errors to occur for the `SceneInstaller` class because the newly defined assembly does not reference the Zenject assembly. The Zenject plugin folder also contains an assembly definition, named `zenject`, and in the `Infrastructure` project add an Assembly Definition Reference to the `zenject` project in order to handle these errors. In the current version of Unity, you must select a different file in the Project tree for the Assembly Definition changes to be applied. If you are prompted to apply the "Unapplied import settings", hit the Apply button.
```

Reply:
Not sure what this suggested change is? Looks identical to what's there, or am I missing something?

Reply:
The change highlighting really leaves something to be desired in this case. Second last sentence, sixth last word, type: Assebmly should be Assembly.

Reply:
Hmmm, I actually completely removed that line, but I guess GitHub doesn't update code in the Conversation tab that has been commented on. If you take a look at the Files Changed tab you should see the updated changes I applied after your first comment.